validator/client/validator.go

```diff
@@ -705,7 +705,8 @@ func (v *validator) RolesAt(ctx context.Context, slot primitives.Slot) (map[[fie
 
 			aggregator, err := v.isAggregator(ctx, duty.Committee, slot, bytesutil.ToBytes48(duty.PublicKey), duty.ValidatorIndex)
 			if err != nil {
-				return nil, errors.Wrap(err, "could not check if a validator is an aggregator")
+				aggregator = false
+				log.WithError(err).Error("Could not check if a validator is an aggregator")
```

Comment:
Tiny nit since it's checking a specific validator
```suggestion
				log.WithError(err).Error("Could not check if the validator is an aggregator")
```